src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.java

```diff
@@ -187,9 +187,9 @@ public static DefaultShardManagerBuilder createDefault(@Nullable String token)
      * @param  token
      *         The bot token to use
      * @param  intent
-     *         The intent to enable
+     *         The intent to enable. This will override the {@link GatewayIntent#DEFAULT default Intents} used by this method
```

Comment:
This should be in the description and a better wording is:

> You can omit intents in this method to use GatewayIntent.DEFAULT and enable additional intents with enableIntents(...)

Reply:
Where would the comment fit the best?
As a `<br>` under the top one or better as its own paragraph?

Reply:
Doesn't matter to me

Reply:
I decided to make it a own paragraph underneath the list.